articles/quickstart/webapp/aspnet-core/files/Program.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Program.cs
+language: csharp
+---
+
+```csharp
+builder.Services.AddControllersWithViews();
+builder.Services.AddAuth0WebAppAuthentication(options =>
+{
+    options.Domain = builder.Configuration["Auth0:Domain"];
+    options.ClientId = builder.Configuration["Auth0:ClientId"];
+});
+
+app.UseAuthentication();
+app.UseAuthorization();
+```
```

Comment:
I assume we want to remove https://github.com/auth0/docs/blob/master/articles/quickstart/webapp/aspnet-core/files/startup.md ?

Reply:
We will also need to remove it from the files array in the frontmatter on interactive.md, and ensure to add the `files/program` file instead.

Reply:
No worries, I am happy to try this once reviewed and make any adjustments we missed.